teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Teacher < person
+  def initialize(_classroom, age, specialization, name = 'unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly remove `_classroom` from here as it has nothing to do with this class.

- [ ] Kindly ensure you are consistent with your parameters. In the Person class, which Teacher inherits from, `parent_permission` is assigned using the equal sign (=), but here you are using the colon notation (:). Kindly choose one and be consistent.